notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<p>Remove this whole section about BUCKET_NAME since it's not needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>